docs/source/getting-started/intro-quickstart/index.rst

```diff
@@ -0,0 +1,229 @@
+MLflow Tracking Quickstart
+==========================
+
+Welcome to MLflow! 
+
+The purpose of this quickstart is to provide a quick, no-nonsense, bare-bones guide to the most essential core APIs of MLflow Tracking. 
+Specifically, those that enable the logging, registering, and loading of a model for inference. 
+
+.. note::
+    For a more in-depth and tutorial-based approach (if that is your style), please see the 
+    `Getting Started with MLflow <../logging-first-model/index.html>`_ tutorial. We recommend that you start here first, though, as this quickstart 
+    uses the most common and frequently-used APIs for MLflow Tracking and serves as a good foundation for the other tutorials in the documentation. 
+
+What you will learn
+-------------------
+
+In just a few minutes of following along with this quickstart, you will learn:
+
+* How to **log** parameters, metrics, and a model
+* The basics of the **MLflow fluent API**
+* How to **register** a model during logging 
+* How to navigate to a model in the **MLflow UI** 
+* How to **load** a logged model as a PyFunc for inference 
```

Comment:
I am not sure if people care about the concept "PyFunc" at model loading. Shall we instead say "How to load a logged model for inference"?

Reply:
removed